### PR TITLE
CASMINST-6066: BOS export/import: Linting

### PR DESCRIPTION
**Pull request by mharding-hpe**
# Description

Linting (and other minor improvements) of the BOS import/export procedure, ahead of more substantial changes, so that the upcoming PRs can be focused just on the technical changes to the procedures.

Note that I've talked with @jsollom-hpe and we've also identified some problems with the current BOS import/export procedure and scripts. However, I still want to keep this PR to just be linting, without addressing those issues. I will address those issues in a subsequent PR.

Backports:
- [1.4 backport](https://github.com/Cray-HPE/docs-csm/pull/3432)
- [1.3 backport](https://github.com/Cray-HPE/docs-csm/pull/3433)

No backports needed for earlier than 1.3

# Checklist

- [X] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [X] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [X] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

 
[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

**Review by rustydb — APPROVED** [review]

File: operations/boot_orchestration/BOS_export_import.md [hunk 1]
```diff
@@ -1,66 +1,174 @@
...
+      ```
+
+   1. Write the contents of the session template to a file in the export directory created earlier.
+
+      > This command strips the `name` field from the session template, because the name is stored in the name of the JSON file itself,
+      > and because that field must be stripped before doing the import.
+
+      ```bash
+      cray bos v2 sessiontemplate describe "${BOS_TEMPLATE_NAME}" --format json |
+          jq 'del(.name)' |
```
Comment (mharding-hpe) [inline operations/boot_orchestration/BOS_export_import.md:98]:
```suggestion
      ```bash
      cray bos v2 sessiontemplate describe "${BOS_TEMPLATE_NAME}" --format json |
```

File: operations/boot_orchestration/BOS_export_import.md [hunk 2]
```diff
@@ -1,66 +1,170 @@
...
+            "path": "s3://boot-images/c23f3d5e-223a-4fb9-b305-0c2be8e63615/manifest.json",
+            "rootfs_provider": "cpss3",
+            "rootfs_provider_passthrough": "dvs:api-gw-service-nmn.local:300:nmn0",
+            "type": "s3"
+          }
+        },
+        "cfs": {
+          "configuration": "uan-config-2.0.0"
+        },
+        "enable_cfs": true
```
Comment (mharding-hpe) [inline operations/boot_orchestration/BOS_export_import.md:133]:
```suggestion
        "boot_sets": {
          "uan": {
            "boot_ordinal": 2,
            "kernel_parameters": "console=ttyS0,115200 bad_page=panic crashkernel=340M hugepagelist=2m-2g intel_iommu=off intel_pstate=disable iommu=pt ip=nmn0:dhcp numa_interleave_omit=headless numa_zonelist_order=node oops=panic pageblock_order=14 pcie_ports=native printk.synchronous=y quiet rd.neednet=1 rd.retry=10 rd.shell turbo_boost_limit=999 ifmap=net2:nmn0,lan0:hsn0,lan1:hsn1 spire_join_token=${SPIRE_JOIN_TOKEN}",
            "network": "nmn",
            "node_list": [
              "x3000c0s15b0n0"
            ],
            "path": "s3://boot-images/c23f3d5e-223a-4fb9-b305-0c2be8e63615/manifest.json",
            "rootfs_provider": "cpss3",
            "rootfs_provider_passthrough": "dvs:api-gw-service-nmn.local:300:nmn0",
            "type": "s3"
          }
        },
        "cfs": {
          "configuration": "uan-config-2.0.0"
        },
        "enable_cfs": true,
        "name": "uan-sessiontemplate-2.0.27"
```

**Review by nrockershousen — APPROVED** [review]

**Review by jsollom-hpe — APPROVED** [review]

**Review by jsl-hpe — APPROVED** [review]

File: .spelling [hunk 3]
```diff
@@ -892,6 +892,9 @@ Lorem
 # To allow Site Init
 Init
 
+- operations/boot_orchestration/BOS_export_import.md
+Redis
```
Comment (rustydb) [inline .spelling:896]: I'm doing a sweep to reorganize the `.spelling` file, application names will be moved to the global list.

Do you think within this PR that "Redis" could just be moved to the global list?

Comment (mharding-hpe) [reply .spelling]: Done.

**Pull request merged**